dev/node_app/src/app.ts

```diff
@@ -122,6 +122,26 @@ type FormValues = {
     message: string;
 }
 
+app.post('/send', (req, res) => {
+    res.json({
+        type: 'form',
+        form: {
+            title: 'Hello, world!',
+            icon: 'icon.png',
+            fields: [
+                {
+                    type: 'text',
+                    name: 'message',
+                    label: 'message',
+                },
+            ],
+            submit: {
+                path: '/send/submit',
+            },
+        },
+    });
+});
```

Comment:
I would prefer not to duplicate this form, to drive home the fact that the same form can be reused across commands and modals. One of my suggestions above makes it so this will actually no longer be called anyway